client/components/signup-form/index.jsx

```diff
@@ -461,7 +485,6 @@ class SignupForm extends Component {
 		return (
 			<p className="signup-form__terms-of-service-link">{ tosText }</p>
 		);
-
```

Comment:
Did you mean to include these changes with this PR? This looks eerily similar to https://github.com/Automattic/wp-calypso/pull/1785.